<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogmatch</class>
 <widget class="QDialog" name="Dialogmatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Statistique">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>570</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(228, 183, 255);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius:14;
font-size: 20px;

}</string>
   </property>
   <property name="text">
    <string>Statistique</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifierM">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(228, 183, 255);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius:14;
font-size: 20px;

}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icones/icones/modifier.png</normaloff>:/icones/icones/modifier.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="id_match">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_MATCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="Minute">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabMatch">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>661</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background_color:rgb(198, 198, 198);
couleur:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_minute">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Minute_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomMatch_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SupprimerM">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(228, 183, 255);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius:14;
font-size: 20px;

}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icones/icones/supp.png</normaloff>:/icones/icones/supp.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="Equipe_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomMatch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_match_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_MATCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AjouterM">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(228, 183, 255);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius:14;
font-size: 20px;

}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icones/icones/ajout.png</normaloff>:/icones/icones/ajout.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="nom_joueur_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Joueur</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
   <item>
    <property name="text">
     <string>But</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carton Jaune</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carton Rouge</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_idM_mod">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_equipe_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_equipe">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_match_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_MATCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="Type_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="Type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idmatch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
   <item>
    <property name="text">
     <string>But</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carton Jaune</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carton Rouge</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_idM_Supp">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>320</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(228, 183, 255);
color : rgb(0, 0, 0);

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_minute_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nom_joueur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Joueur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
